sdk/digitaltwins/azure-digitaltwins-core/src/test/java/com/azure/digitaltwins/core/ModelsTestBase.java

```diff
@@ -0,0 +1,83 @@
+package com.azure.digitaltwins.core;
+
+import com.azure.core.http.HttpClient;
+import com.azure.digitaltwins.core.models.ModelData;
+import org.junit.jupiter.api.Test;
+import org.opentest4j.AssertionFailedError;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * This abstract test class defines all the tests that both the sync and async model test classes need to implement. It also
+ * houses some model test specific helper functions.
+ */
+public abstract class ModelsTestBase extends DigitalTwinsTestBase {
+    @Test
+    public abstract void modelLifecycleTest(HttpClient httpClient, DigitalTwinsServiceVersion serviceVersion);
+
+    @Test
+    public abstract void getModelThrowsIfModelDoesNotExist(HttpClient httpClient, DigitalTwinsServiceVersion serviceVersion);
+
+    @Test
+    public abstract void createModelThrowsIfModelAlreadyExists(HttpClient httpClient, DigitalTwinsServiceVersion serviceVersion);
+
+    @Test
+    public abstract void getModelThrowsIfModelIdInvalid(HttpClient httpClient, DigitalTwinsServiceVersion serviceVersion);
+
+    static void createModelsRunner(String buildingModelId, String floorModelId, String hvacModelId, String wardModelId, Consumer<List<String>> createModelsTestRunner) {
+        String modelBuilding = TestAssetsHelper.getBuildingModelPayload(buildingModelId, hvacModelId, floorModelId);
+        String modelHvac = TestAssetsHelper.getHvacModelPayload(hvacModelId, floorModelId);
+        String modelWard = TestAssetsHelper.getWardModelPayload(wardModelId);
+
+        List<String> modelsList = new ArrayList<>();
```

Comment:
you can do 
`List<String> modelsList = new ArrayList<>(Arrays.asList(modelBuilding, modelHvac, modelWard))`